### DISCUSSION TITLE
Drawer issues with Firefox, please help

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#1408] [Q&A] [answered]
Hi

My website behaves differently for the browsers, https://sithasodaisyui.netlify.app/

This is my drawer on Firefox..

![image](https://user-images.githubusercontent.com/5799306/204991405-a58a61fb-aa37-4992-990e-8ff615214ee4.png)

The same drawer on Brave.

![image](https://user-images.githubusercontent.com/5799306/204991476-90dd7a28-e504-40a4-b473-967c164e83ba.png)

I cant seem to fix the x-scroll at all, tried the overflow classes.

Can someone please advice, what am I missing.

This is the only other code I have

``
::-webkit-scrollbar {
background: transparent;
width: 12px !important;
}
::-webkit-scrollbar-thumb {
background: #606060 !important;
border-radius: 100px !important;
}``

Thanks in advance.

**Comment by saadeghi** [comment, marked as answer]
Fixed in `2.45.0`
